_build/blizzard.j

```diff
@@ -1,10254 +0,0 @@
-//===========================================================================
```

Comment:
What impact does it have removing blizzard.j and common.j? Are those files automatically fetched by the compiler? Maybe they belong in .wurst then?

Reply:
they are

Reply:
Okay, I'm happy to remove these. I have no interest in supporting anything other than the latest stable battle.net patch.